docs/source/intro.rst

```diff
@@ -0,0 +1,122 @@
+
+
+
+Features
+========
+
+PyMC3 strives to make Bayesian modeling as simple and painless as possible, thereby allowing users to focus on their scientific problem, rather than on the methods used to solve it. Here is a partial list of its features:
+
+* Fits Bayesian statistical models using modern techniques, including MCMC and VI.
+
+* Includes a large suite of well-documented statistical distributions.
+
+* Uses Theano as the computational backend, allowing for fast expression evaluation, automatic gradient calculation, and GPU computing.
+
+* Includes a module for Gaussian process modeling.
+
+* Creates summaries including tables and plots.
+
+* Several convergence diagnostics are available.
+
+* Extensible: easily incorporates custom step methods and unusual probability
+  distributions.
+
+* Bayesian models can be embedded in larger programs, and results can be analyzed
+  with the full power of Python.
+
+
+What's new in version 3
+=======================
+
+The third major version of PyMC has benefitted from being re-written from scratch. Substantial improvements in the user interface and performance have resulted from this. While PyMC2 relied on Fortran extensions (via f2py) for most of the computational heavy-lifting, PyMC3 leverages Theano, a library from the Montréal Institute for Learning Algorithms (MILA), for array-based expression evaluation, to perform its computation. What this provides, above all else, is fast automatic differentiation, which is at the heart of the gradient-based sampling and optimization methods currently providing inference for probabilistic programming. 
+
+Most notably, the PyMC3 provides:
+
+* New flexible object model and syntax (not backward-compatible with PyMC2).
+
+* Gradient-based MCMC methods, including Hamiltonian Monte Carlo (HMC), the No U-turn Sampler (NUTS), and Stein Variational Gradient Descent.
+
+* Variational inference methods, including automatic differentiation variational inference (ADVI) and operator variational inference (OPVI). 
+
+* An interface for easy formula-based specification of generalized linear models (GLM).
+
+* New elliptical slice sampler method.
+
+* Specialized distributions for representing time series.
+
+* A library of Jupyter notebooks that provide case studies and fully developed usage examples.
+
+* Much more!
+
+While the addition of Theano adds a level of complexity to the development of PyMC, fundamentally altering how the underlying computation is performed, we have worked hard to maintain the elegant simplicity of the original PyMC model specification syntax. 
+
+Usage Overview
+==============
+
+First, import the PyMC3 functions and classes you will need for building your model. You can import the entire module via `import pymc3 as pm`, or just bring in what you need::
+
+    from pymc3 import Model, Normal, invlogit, Binomial, sample, traceplot
```

Comment:
Pretty much everywhere we are using `import pymc3 as pm`. So I think we should do the same thing here.